<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ayrinti2</class>
 <widget class="QDialog" name="Ayrinti2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2" rowspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="5">
    <widget class="QPushButton" name="btnEkle">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="6">
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="btnCikar">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QPushButton" name="btnKaydet">
     <property name="text">
      <string>Kaydet</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="btnIptal">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
